### PR TITLE
Commander always syncs from initial block when bootstrapping via remote node URL

### PR DESCRIPTION
**Pull request by max-sidorov**
> Previous ChainState stored in DB is always overridden by fetched ChainState

### CODE REVIEW

File: commander/commander.go [hunk 1]
```diff
@@ -295,7 +296,16 @@ func bootstrapFromRemoteState(
 		return nil, errors.WithStack(errInconsistentRemoteChainID)
 	}
 
-	return setGenesisStateAndCreateClient(blockchain, storage, fetchedChainState, cfg.Rollup, commanderMetrics)
+	if dbChainState == nil {
+		return setGenesisStateAndCreateClient(blockchain, storage, fetchedChainState, cfg.Rollup, commanderMetrics)
+	}
+
+	if !fetchedChainState.Equal(dbChainState) {
```
Comment (Marik-D) [inline commander/commander.go:303]: Do chain states here include "syncedBlock" field? Would it cause problems with the comparison here?

File: commander/commander.go [hunk 2]
```diff
@@ -233,7 +233,7 @@ func getClient(
 	}
 	if cfg.Bootstrap.BootstrapNodeURL != nil {
```
Comment (Marik-D) [inline commander/commander.go:234]: I think If we already have chain state in db, we dont need to check with the remote mode. Just ignore the URL in the config and initialize from the saved state.

File: commander/commander.go (hunk 1 above)
Comment (max-sidorov) [reply commander/commander.go:303]: There is a field `SyncedBlock`, but it is not compared in the [ChainState.Equal](https://github.com/worldcoin/hubble-commander/blob/239611e4ab05a5cdef22bcdb20063d637003ddc7/models/chain_state.go#L38-L64) method.

**Comment by max-sidorov** [comment]
Now ChainState from the database is used first. If the database is empty, the spec file or remote node is used.

**Review by Marik-D — APPROVED** [review]

**Review by msieczko — APPROVED** [review]

**Pull request merged**